docs/user/getting-startedv2.rst

```diff
@@ -0,0 +1,116 @@
+.. _getting_startedv2:
+
+Getting Started V2
+==================
+This section takes you through a the entire process of conducting a co-simulation with Maestro2.
+
+| This guide is based on the command line interface (CLI) of Maestro2 and consists of two parts: 
+| The first part concerns writing a MaBL Specification by hand and executing it.
+| The second part concerns using the specification generation and expansion capabilities of Maestro.
+
+Note that Maestro2 also features a web API presented in :ref:`web-api`.
+
+Setup
+-----
+The first step is to install the dependencies along with the tools and resources that are to be used
+
+- Download and install Java 11
+- Download the :download:`example resources part1 <images/wt_example/watertank-example-part1.zip>` and unzip into a directory.
+- Download the :download:`example resources part2 <images/wt_example/watertank-example-part2.zip>` and unzip into a directory.
+- Download `maestro-2.x.x-jar-with-dependencies.jar` from the newest `Maestro2 2.x.x` release on the Maestro release page https://github.com/INTO-CPS-Association/maestro/releases and place it in the same directory as the example resources were unzipped into.
+
+.. _getting_started_part1:
+
+Part 1: First MaBL Specification
+------------------------
+The example below concerns a water tank. The tank (Continuous Time component) has a constant inflow of water and a valve that is controlled by a controller (Discrete-Event component). When the valve is open, the water level within the tank decreases, and whe the valve is closed, the water level increases.
+The controller determines when to open and close the valve based on a maximum and a minimum water level.
+
+.. _fig-watertankexample:
+.. figure:: images/wt_example/wt-example.png
+    :align: center
+
+    Water Tank Example
+
+The corresponding MaBL code, including descriptive comments, to execute this co-simulation is presented below:
+
+.. literalinclude:: images/wt_example/wt-example.mabl
+    :language: c
+
+To execute this (one can use the file corresponding wt-example.mabl from the unzipped example resouces) run the following command from the terminal:
+
+.. code-block:: none
+
+    > java -jar maestro-2.x.x-jar-with-dependencies.jar --interpret wt-example.mabl
+        where --interpret is to interpret a specification
```

Comment:
I would probably prefer just to have the valid command here so I could copy it. Leaving out the "where" part. The above text could just be changed to say evaluate/interpret it or something like that

Reply:
Fixed